azure-boards-common/src/main/java/com/synopsys/integration/azure/boards/common/oauth/AzureTokenResponse.java

```diff
@@ -38,67 +38,6 @@ public AzureTokenResponse() {
         super();
     }
 
-    @Override
```

Comment:
The setters and getters here aren't needed for the  deserialization to work properly.  The annotation is the most important part.  Clean up this class to the bare minimum.